libmscore/edit.cpp

```diff
@@ -4666,6 +4666,26 @@ void Score::undoAddElement(Element* element)
             }
       }
 
+//---------------------------------------------------------
+//   prevInstrumentChange
+//---------------------------------------------------------
+
+InstrumentChange* Score::prevInstrumentChange(Segment* e, int track)
```

Comment:
not a massive problem, but maybe better naming for the segment variable than `e`? `e` usually refers to an element, `s` or even `seg` would be better, I think.

Reply:
Ah, thank you for spotting this. It was left over from a previous version which took an element as a parameter, but I'll change that now